manifests/init.pp

```diff
@@ -115,6 +118,7 @@
   Stdlib::Fqdn $servername = $facts['networking']['fqdn'],
   Array[Stdlib::Absolutepath] $allowed_import_path = ['/var/lib/pulp/sync_imports'],
   Optional[String] $remote_user_environ_name = undef,
+  Integer[1, 24] $worker_count = min(8, $facts['processors']['count']),
```

Comment:
Is a worker per CPU recommended by Pulp team?

Reply:
@jlsherrill thoughts on this, or do you know who from the Pulp team would be able to answer? The default behavior I have set here is the same as what we did with Pulp2 but perhaps there are different considerations for Pulpcore.

Reply:
We were discussing this just the other day actually. We need to seek guidance from the Pulp community. @wbclark would you like to send an email to pulp-list@redhat.com to ask about Pulp 3 worker tuning?

Reply:
Should the second number be 8? `Integer[1, 8]`